ports/stm32/boards/NUCLEO_F746ZG/mpconfigboard.mk

```diff
@@ -4,3 +4,6 @@ AF_FILE = boards/stm32f746_af.csv
 LD_FILES = boards/stm32f746.ld boards/common_ifs.ld
 TEXT0_ADDR = 0x08000000
 TEXT1_ADDR = 0x08020000
+
+# MicroPython settings
+MICROPY_PY_LWIP = 1
```

Comment:
Following other boards with ETH, I think it'd be good to also enable the `ussl` module via:
```
MICROPY_PY_USSL = 1
MICROPY_SSL_MBEDTLS = 1
```

Reply:
It sounds good, I will add it

Reply:
> Following other boards with ETH, I think it'd be good to also enable the `ussl` module via:
> 
> ```
> MICROPY_PY_USSL = 1
> MICROPY_SSL_MBEDTLS = 1
> ```

Done